.azure-pipelines/generation-pipeline.yml

```diff
@@ -196,8 +258,14 @@ stages:
 
 - stage: stage_php_v1
   dependsOn:
+  - stage_build_and_publish_typewriter
   - stage_v1_metadata
```

Comment:
I assume this doesn't block if `stage_v1_metadata` is skipped, i.e. `condition` below overrules this.

Reply:
Correct, the condition overrules and helps with flow control. nit: the diagramming tool uses the dependsOn too to show the relationships between stages. I bet that if we group stacked the stages in the yaml based on the metadata stage dependency it would show a lot better in the diagram.